components/smee/staging/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Maybe we should create all those resources in their own namespace, i.e not the default one? If you agree, then put namespace in this file and it will be applied to all resources.
```
namespace: someNamespace
```

Reply:
humm, that is a good point. Let me try this and confirm is this is the case.

Reply:
I just tested this and you are correct, having the namespace at the application level is enough. You can ignore this comment and sorry for the noise :)